wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md

```diff
@@ -26,9 +29,9 @@ Hard Method (keep local changes):
 4.  Go into editor. Make sure the "Web" menu is enabled (not greyed out).
 5.  Change the Artist/Title info.
 6.  **Do NOT exit the editor.**
-7.  Use **File &gt; Open...** to change the Artist/Title in other difficulties.
+7.  Use **File > Open...** to change the Artist/Title in other difficulties.
 8.  Full Submit.
 
 If you have any duplicated maps, please contact a BAT for resolution.
 
-<span style="color:#FF4000;">**DO NOT SUBMIT IF THE EDITOR "WEB" MENU IS DISABLED. YOUR MAP WILL DUPLICATE.**</span>
+**DO NOT SUBMIT IF THE EDITOR "WEB" MENU IS DISABLED. YOUR MAP WILL DUPLICATE.**
```

Comment:
`**Do not submit if the editor "web" menu is disabled. Your map will duplicate.**`